mne/viz/_brain/_brain.py

```diff
@@ -1023,6 +1027,100 @@ def update_auto_scaling(self, restore=False):
                     _set_colormap_range(actor, ctable, scalar_bar, rng)
                     self._data['ctable'] = ctable
 
+    def save_movie(self, filename, time_dilation=4., tmin=None, tmax=None,
+                   framerate=24, interpolation='quadratic', codec=None,
+                   bitrate=None, **kwargs):
+        """Save a movie (for data with a time axis).
+
+        The movie is created through the :mod:`imageio` module. The format is
+        determined by the extension, and additional options can be specified
+        through keyword arguments that depend on the format. For available
+        formats and corresponding parameters see the imageio documentation:
+        http://imageio.readthedocs.io/en/latest/formats.html#multiple-images
+
+        .. Warning::
+            This method assumes that time is specified in seconds when adding
+            data. If time is specified in milliseconds this will result in
+            movies 1000 times longer than expected.
+
+        Parameters
+        ----------
+        filename : str
+            Path at which to save the movie. The extension determines the
+            format (e.g., `'*.mov'`, `'*.gif'`, ...; see the :mod:`imageio`
+            documenttion for available formats).
+        time_dilation : float
+            Factor by which to stretch time (default 4). For example, an epoch
+            from -100 to 600 ms lasts 700 ms. With ``time_dilation=4`` this
+            would result in a 2.8 s long movie.
+        tmin : float
+            First time point to include (default: all data).
+        tmax : float
+            Last time point to include (default: all data).
+        framerate : float
+            Framerate of the movie (frames per second, default 24).
+        interpolation : str
+            Interpolation method (``scipy.interpolate.interp1d`` parameter,
+            one of 'linear' | 'nearest' | 'zero' | 'slinear' | 'quadratic' |
+            'cubic', default 'quadratic').
+        **kwargs :
+            Specify additional options for :mod:`imageio`.
+        """
+        from scipy.interpolate import interp1d
+        from math import floor
+
+        try:
+            import imageio
+            has_imageio = True
+        except (ImportError, ModuleNotFoundError):
+            has_imageio = False
+            warn("The imageio module is not found, "
+                 "save_movie() will return the image sequence instead")
```

Comment:
I wouldn't do this. Let's just raise an error, say that it's required for movie saving, and add to requirements.txt/environment.yml/README.rst